### PR TITLE
[REF] auto-fix conventions

### PR DESCRIPTION
**Pull request by moylop260**

### CODE REVIEW

**Comment by dreispt** [comment]
:+1:
Will be green when https://github.com/OCA/maintainer-quality-tools/pull/136 is merged.

File: mgmtsystem_audit/wizard/copy_verification_lines.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 
 class copy_verification_lines(orm.TransientModel):
+
     """
     Copy Verification Lines
     """
```
Comment (bwrsandman) [inline mgmtsystem_audit/wizard/copy_verification_lines.py:28]: :-1: Is the newline before a doctring a positive change to style?

File: mgmtsystem_claim/__init__.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # -*- encoding: utf-8 -*-
-import mgmtsystem_claim
+from . import mgmtsystem_claim
 # vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (bwrsandman) [inline mgmtsystem_claim/__init__.py]: Please remove `#vim` lines while you're there

File: mgmtsystem_kpi/mgmtsystem_kpi.py [hunk 3]
```diff
@@ -60,6 +60,7 @@ def is_select_query(query):
 
 
 class mgmtsystem_kpi_category(orm.Model):
+
     """
     KPI Category
     """
```
Comment (bwrsandman) [inline mgmtsystem_kpi/mgmtsystem_kpi.py:65]: :-1: Is the newline before a doctring a positive change to style?

File: mgmtsystem_kpi/mgmtsystem_kpi.py [hunk 4]
```diff
@@ -72,6 +73,7 @@ class mgmtsystem_kpi_category(orm.Model):
 
 
 class mgmtsystem_kpi_threshold_range(orm.Model):
+
     """
     KPI Threshold Range
     """
```
Comment (bwrsandman) [inline mgmtsystem_kpi/mgmtsystem_kpi.py:77]: :-1: Is the newline before a doctring a positive change to style?

File: mgmtsystem_kpi/mgmtsystem_kpi.py [hunk 5]
```diff
@@ -225,6 +227,7 @@ def _generate_invalid_message(
 
 
 class mgmtsystem_kpi_threshold(orm.Model):
+
     """
     KPI Threshold
     """
```
Comment (bwrsandman) [inline mgmtsystem_kpi/mgmtsystem_kpi.py:230]: :-1: Is the newline before a doctring a positive change to style?

File: mgmtsystem_kpi/mgmtsystem_kpi.py [hunk 6]
```diff
@@ -329,6 +332,7 @@ def get_color(self, cr, uid, ids, kpi_value, context=None):
 
 
 class mgmtsystem_kpi_history(orm.Model):
+
     """
     History of the KPI
     """
```
Comment (bwrsandman) [inline mgmtsystem_kpi/mgmtsystem_kpi.py:334]: :-1: Is the newline before a doctring a positive change to style?

File: mgmtsystem_kpi/mgmtsystem_kpi.py [hunk 7]
```diff
@@ -361,6 +365,7 @@ class mgmtsystem_kpi_history(orm.Model):
 
 
 class mgmtsystem_kpi(orm.Model):
+
     """
     Key Performance Indicators
     """
```
Comment (bwrsandman) [inline mgmtsystem_kpi/mgmtsystem_kpi.py:366]: :-1: Is the newline before a doctring a positive change to style?

**Comment by bwrsandman** [comment]
:-1: The automatic tool needs some work.

Adding a space before a docstring does not seems like it improves style, the docstring should be right up to the module/function/class description.

The tool also ignores vim lines, it would be good to remove them as well, but make sure to take out extra new lines at the end of a file.

**Comment by bwrsandman** [comment]
Revised commit by hand. Should be better now.

**Comment by bwrsandman** [comment]
Note to forward port once merged

**Comment by dreispt** [comment]
:+1:

**Comment by bwrsandman** [comment]
@pedrobaeza please come add your +1

**Comment by pedrobaeza** [comment]
hehe, waiting for Travis...

**Comment by pedrobaeza** [comment]
My :+1: :wink:

**Comment by bwrsandman** [comment]
Forward ported to 8.0 with ad23dae252

**Pull request merged**